### PR TITLE
Feat: Add set_admin route in the authentication gateway

### PR DESCRIPTION
**Pull request by HaroldCuico**
Added a set_admin route in the authenication gateway

### CODE REVIEW

**Review by JMark-FS — APPROVED** [review]
Looks good

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/AuthenticationGatewayController.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Controllers/AuthenticationGatewayController.cs:1]: Good work!
I still have to refactor the issue from the rabbitmq side

**Review by jaymar921 — APPROVED** [review]
Good work master hashira!
Figure

**Comment by jaymar921** [comment]
You can now merge
Figure

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/AuthenticationGatewayController.cs
Comment (HaroldCuico) [reply WebApp/backend/QuizMaster.API.Gatewway/Controllers/AuthenticationGatewayController.cs:1]: G

**Pull request merged**